pages/common/java.md

```diff
@@ -1,12 +1,16 @@
 # java
 
 > Java Application Launcher.
-> More information: <https://java.com>.
+> More information: <https://www.oracle.com/java/technologies/javase-documentation.html>.
```

Comment:
I think this is a better reference for the command-line
```suggestion
> More information: <https://docs.oracle.com/en/java/javase/17/docs/specs/man/java.html>.
```

Reply:
I chose the root page of the Java SE documentation over the Java 17 page as Java changes versions every 6 months and that link would be outdated as quick as the PR would be accepted. That said, I'm open to changing it if that is the consensus.

Reply:
I understand, and yes it will be outdated. but it's hard to find that with the link that you provide.